fizzbuzz.js

```diff
@@ -0,0 +1,26 @@
+const fizzbuzz = (number) => {
+  if ((number % 3 === 0) && (number % 5 === 0)) {
+      return 'fizzbuzz';
+  } else if (number % 3 === 0) {
+      return 'fizz';
+  } else if (number % 5 === 0) {
+      return 'buzz';
+  }
+      return number;
+};
+    
+const main = () => {
+  for (var i = 1; i <= 100; i++) {
```

Comment:
choose either a for loop or a for...of loop, and then use it consistently throughout your code. Here, you're using a for loop; below, you've used a for...of loop.

Reply:
Sorry, this is my fault. I'll cut it out.